CONTRIBUTING.md

```diff
@@ -30,6 +30,11 @@ Alternatively, you can use `npm run dev` if you don't want tests to be run
 
 To run the tests as they are being executed in the CI, run `npm run test:prod`.
 
+### Testing SaveSchemaState
+
+If you change the persisting of the model in [SaveSchemaState.ts](/frontend/src/model/schema/methodObjects/SaveSchemaState.ts) and other schema files you always have to update the zip files savedExampleSchema.zip and savedSchema.zip in cypress forlder [Fixtures](/cypress/fixtures) for tests [load-saved-schema.cy.ts](/cypress/e2e/frontend/components/load-saved-schema.cy.ts) and [tables-selection.cy.ts](/cypress/e2e/frontend/pages/tables-selection.cy.ts).
+The schema savedSchema.zip just includes the two testing tables without any schema transformations. The schema of savedExampleSchema.zip should be include lots of your possible persited schema transformations (like one split, join, union).
```

Comment:
Der zweite Satz ist grammatikalisch komisch, und man müsste vielleicht noch erklären was "just the two test tables" heißt. Kann heute noch mittag auch selbst nen Vorschlag machen.